apps/filecoin-site/src/app/_components/LinkCard.tsx

```diff
@@ -12,6 +12,7 @@ type LinkCardProps = {
   as: 'li' | 'article' | 'div'
   backgroundVariant: SectionProps['backgroundVariant']
   title: string
+  titleAs?: 'h2' | 'h3'
```

Comment:
Can we use `headingTag: HeadingProps['tag']` like in `apps/filecoin-site/src/app/_components/SectionSubContent.tsx`?

Reply:
I apologise! Can you maybe check again 🙏 I made changes but maybe I misunderstood what you meant

Reply:
All good now!